101-azure-machine-learning-service-workspace/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
format the json (e.g. SHIFT+ALT+F in VS Code)

Reply:
done